tools/gn.rs

```diff
@@ -110,7 +110,7 @@ impl Build {
         .unwrap();
       ninja
         .env("PYTHONPATH", python_path.join(";"))
-        .env("PATH", path + &orig_path)
+        .env("PATH", path + ";" + &orig_path)
```

Comment:
this is wrong btw

Reply:
The semi-colon is added to orig_path above. It is not elegant but is it incorrect? (Or am I misunderstanding?)

Reply:
Yeah I added it, then realized it was done above and thus unnecessary.